config/routes.rb

```diff
@@ -473,6 +473,14 @@
     end
   end
 
+  resources :lti, only: [] do
```

Comment:
Ah, please try using `namespace` instead of `resources` (https://guides.rubyonrails.org/routing.html#controller-namespaces-and-routing). I think that should simplify the path helper too